__tests__/Footer.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act } from 'react-dom/test-utils';
+
+import Footer from '../src/components/Footer';
+
+let container = null;
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  unmountComponentAtNode(container);
+  container.remove();
```

Comment:
what's the purpose of unmounting and removing()? Is this to ensure everything gets removed from container?

Reply:
If you don't perform clean-up and unmount the tree from the document, tests can become "leaky" and one test can change the behavior of another test. That makes them difficult to debug.